Style.css

```diff
@@ -0,0 +1,3 @@
+h2 {
+    color:greenyellow;
+}
```

Comment:
kindly add the ending line so that your codes look good.

Reply:
- [ ] Please this comment is not clear. What ending line are you talking about ?